functions-and-scope.js

```diff
@@ -70,6 +116,24 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
 // Log het antwoord in de terminal.
 
+function highestGrade(highestNumber) {
+
+    let high = 0
```

Comment:
Het is altijd beter om te beginnen met de eerste waarde van de array. Beetje flauw bij dit voorbeeld, maar stel dat je een array hebt van `[-2, -5, -1, -3]`. Als nu deze array aan jouw functie mee zou geven, dan komt daar `0` uit, terwijl dat cijfer helemaal niet in de array voorkomt.

```suggestion
    let high = highestNumber[0];
```